homeassistant/components/vodafone_station/sensor.py

```diff
@@ -23,25 +23,42 @@
 from .coordinator import VodafoneStationRouter
 
 NOT_AVAILABLE: list = ["", "N/A", "0.0.0.0"]
+UPTIME_DEVIATION = 30
 
 
 @dataclass(frozen=True, kw_only=True)
 class VodafoneStationEntityDescription(SensorEntityDescription):
     """Vodafone Station entity description."""
 
-    value: Callable[[Any, Any], Any] = (
-        lambda coordinator, key: coordinator.data.sensors[key]
+    value: Callable[[Any, Any, Any], Any] = (
```

Comment:
We know what the types are, in a followup, can we improve the typing here?

Reply:
I think we know only some of them, not all.
Will review carefully anyway.

Reply:
We know all of them, coordinator, statetype, str and return type is also statetype

Reply:
#129143